### ISSUE TITLE
Placeholder not emptying on DraftJS homepage

### ISSUE THREAD

**Issue by stoplion** [facebook/draft-js#546]
Placeholder not emptying on [DraftJS](https://facebook.github.io/draft-js/) homepage.
![image](https://cloud.githubusercontent.com/assets/231343/16934287/9eaacb52-4d09-11e6-806b-d7c2daa3e006.png)
Chrome  51.0.2704

```
react.js:19287 Warning: Each child in an array or iterator should have a unique "key" prop. Check the render method of `InlineStyleControls`. See https://fb.me/react-warning-keys for more information.
```

**Comment by SecurityInsanity** [comment]
Hey @stoplion This sounds like the `-webkit-user-modify` issue. 
Can you test to see if: https://github.com/facebook/draft-js/pull/545 fixes it for you?

**Comment by spicyj** [comment]
Should be fixed by #545.
